packages/analytics-js/src/components/configManager/util/validate.ts

```diff
@@ -64,6 +69,11 @@ const getDataServiceUrl = (endpoint: string, useExactDomain: boolean) => {
   return `${url}/${formattedEndpoint}`;
 };
 
+const isTopLevelDomain = (providedDomain: string): boolean => {
```

Comment:
The utility name is misleading.

Let's rename it to `isWebpageTopLevelDomain`.

Reply:
Addressed